src/components/SecuredRoute/SecuredRoute.tsx

```diff
@@ -13,4 +13,23 @@ const SecuredRoute = (props: any): JSX.Element => {
   );
 };
 
+let lastActive: number = Date.now();
+
+document.addEventListener('mousemove', () => {
+  lastActive = Date.now();
+});
+
+setInterval(() => {
+  const currentTime = Date.now();
+  const timeSinceLastActive = currentTime - lastActive;
+  const logoutUserTime = 20 * 60 * 1000;
+
+  if (timeSinceLastActive > logoutUserTime) {
+    toast.warn('User not found! kindly relogin as sessison has expired');
```

Comment:
1. This message is from the developer perspective. If a user is logged in and they get a message saying that they are not found, they would think that the application was not working.
1. Does this message have the required language translations?

Reply:
I did change the toast message in my earlier PR but i was redirected to use the existing toast message. But I totally understand the confusion from a user's perspective. I'll correct that.

Most toast notifications doesn't have the required language translations. I think I would create an issue on that or refer to the admin on that issue. @palisadoes